test/init.js

```diff
@@ -153,6 +158,66 @@ test('init: should add a valid source map for css if wished', function(t) {
   }).write(makeFileCSS());
 });
 
+test('init: can replace `identityMap` option with sourcemap.identityMap stream (js file)', function(t) {
+  var file = makeFile();
+
+  function assert(files) {
+    t.ok(files[0], 'should pass something through');
+    t.ok(files[0] instanceof File, 'should pass a vinyl file through');
+    t.ok(files[0].sourceMap, 'should add a source map object');
+    t.equal(String(files[0].sourceMap.version), '3', 'should have version 3');
+    t.equal(files[0].sourceMap.sources[0], 'helloworld.js', 'should add file to sources');
```

Comment:
every property on `sourceMap` is actually an indicator.  The property that describes the parsing that happened is `mappings` but I think it's fine to keep everything.